src/pages/index.html

```diff
@@ -1,25 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Landing Page</title>
-    <link rel="stylesheet" href="../stylesheets/_main.scss" />
-  </head>
-  <body>
-    <!-- header -->
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Landing Page</title>
+  <link rel="stylesheet" href="../stylesheets/_main.scss" />
+</head>
+
+<body>
+  <!-- header -->
+  <div class="container">
     <include src="./layouts/header.html"></include>
 
     <!-- main -->
     <include src="./layouts/main.html"></include>
 
     <!-- footer -->
     <include src="./layouts/footer.html"></include>
+  </div>
+
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/html-inspector/0.8.2/html-inspector.js"></script>
+  <script>
+    HTMLInspector.inspect();
+  </script>
+</body>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/html-inspector/0.8.2/html-inspector.js"></script>
-    <script>
-      HTMLInspector.inspect();
-    </script>
-  </body>
 </html>
```

Comment:
end of line

Reply:
Have you fixed my comment yet?

Reply:
> Have you fixed my comment yet?

![image](https://github.com/DatWithTheWorld/practice_html_css/assets/131734339/b9d48bd1-68b7-45dd-92dc-aefb168a376b)
I have already fixed in my lastest commit, sorry for my mistake.

Reply:
i don't see your resolve in your pr i don't see end of line

Reply:
> i don't see your resolve in your pr i don't see end of line

I resolve it at https://github.com/DatWithTheWorld/practice_html_css/pull/4/commits/3396a2442d5a10396203a4ccb9ca50f7092eaadf